ArcGISRuntimeSDKQt_CppSamples/Layers/BrowseOGCAPIFeatureService/README.metadata.json

```
@@ -0,0 +1,21 @@
+{
+    "category": "TODO",
+    "description": "TODO",
+    "ignore": false,
+    "images": [
+        "screenshot.png"
+    ],
+    "keywords": [
+        "TODO"
+    ],
+    "redirect_from": "",
+    "relevant_apis": [
+        "TODO"
+    ],
+    "snippets": [
+        "BrowseOGCAPIFeatureService.qml",
+        "BrowseOGCAPIFeatureService.cpp",
+        "BrowseOGCAPIFeatureService.h"
+    ],
+    "title": "Browse OGC API Feature Service"
+}
```

Good thing to note - we should always have a blank line at the end of files. Here's [some reasoning for that](https://stackoverflow.com/questions/2287967/why-is-it-recommended-to-have-empty-line-in-the-end-of-a-source-file).
```suggestion
}

```